ui/components/app/signature-request-siwe/index.scss

```diff
@@ -0,0 +1,67 @@
+@import 'signature-request-siwe-header/index';
+@import 'signature-request-siwe-message/index';
+
+.signature-request-siwe {
+  display: flex;
+  flex-direction: column;
+  flex: 1;
+  height: 100%;
+  width: 100%;
+  min-width: 0;
+  background-color: var(--color-background-default);
+  overflow-y: auto;
+  margin-right: auto;
+  margin-left: auto;
+
+  @media screen and (min-width: $break-large) {
+    width: 408px;
+    max-height: 82vh;
+    min-height: 570px;
+    flex: 0 0 auto;
+    border-radius: 8px;
+    box-shadow: 0 0 7px 0 rgba(0, 0, 0, 0.08);
+  }
+
+  &__domain-mismatch-warning {
+    padding: 4px 16px;
+  }
+
+  &__page-container-footer {
```

Comment:
This might have been my bad! Had a momentary lapse of judgement on CSS.. that's embarrassing 🤦

Apologies @skgbafa!

And thanks so much for the catch and the resources @digiwand 🙏 💯

Reply:
All good! 🙏  Honestly, it works fine and it's an existing pattern in our codebase. I think in an app like this, it's like to not cause problems. Thing is, I have run into `!important` takeover battles in the past and I think it's good practice to choose specificity over `!important` when applicable. 

[This StackOverflow answer](https://stackoverflow.com/a/3706876/4053142) has a good mention of when it's okay to use `!important` as well

I appreciate you receiving the feedback/suggestion and getting on board with this too! Thank you! 😊 🤜🏼🤛🏼

Reply:
Addressed! I left in a single `!important` as I was unable to prevent a line from showing any other way. I think the high degree of specificity will prevent collisions.
![Screen Shot 2022-06-07 at 6 24 39 PM](https://user-images.githubusercontent.com/5846858/172493303-e3863949-84cb-4f5f-b96b-7c391a0c0364.png)
![Screen Shot 2022-06-07 at 6 25 58 PM](https://user-images.githubusercontent.com/5846858/172493456-7184dc09-4a20-4f8f-bbcc-d6135a1d57a4.png)

Reply:
Addressed and removed all `!important`. Thank you very much for the assistance @digiwand 😄